CRM/Case/XMLProcessor.php

```diff
@@ -62,12 +62,15 @@ public static function mungeCaseType($caseType) {
   }
 
   /**
+   * @deprecated
+   *
    * @param bool $indexName
    * @param bool $all
    *
    * @return array
    */
   public static function &allActivityTypes($indexName = TRUE, $all = FALSE) {
+    CRM_Core_Error::deprecatedFunctionWarning('allActivityTypes() is deprecated. Use CRM_Case_PseudoConstant::caseActivityType() instead.');
```

Comment:
If you look at the signature of `CRM_Core_Error::deprecatedFunctionWarning` it generates the message for you, just pass it the name of the new method:
```suggestion
    CRM_Core_Error::deprecatedFunctionWarning('CRM_Case_PseudoConstant::caseActivityType');
```

Reply:
Ah ok thanks. I just copy/pasted from somewhere. Will update.